src/components/Resources.js

```diff
@@ -1,5 +1,47 @@
 import React from 'react';
-import { Col, Container, Row } from 'react-bootstrap';
+import { Card, Col, Container, Row } from 'react-bootstrap';
+
+// Define the data needed for the resources cards
+const resources = [
```

Comment:
i think it'd be good to import this from a data file

Reply:
good call, updated